src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -171,6 +171,9 @@ case class WorkflowDescriptor(id: WorkflowId,
 }
 
 object WorkflowDescriptor {
+
+  def buildWorkflowRootPath(rootPath: String, name: String, workflowId: WorkflowId) = s"$rootPath/$name/${workflowId.toString}"
```

Comment:
I remember having an issue without it - `toString` is overriden in `WorkflowId` to return the `UUID.toString`